.gitignore

```diff
@@ -16,3 +16,27 @@ Tools/gawk
 Tools/sbinfo
 DownloadedFiles*
 constants.sh
+
+# new development
+# IDE Settings
+.vscode
+
```

Comment:
what are these?

Reply:
ignore this .vscode folder, as this .vscode may contains some workspace settings for current project (LISAv2 and LISAv3), especially when user change branches. This folder may be taken as something 'new' if not been ignored.